app/src/display/status_screen.c

```diff
@@ -19,6 +20,8 @@ static struct zmk_widget_battery_status battery_status_widget;
 static struct zmk_widget_output_status output_status_widget;
 #endif
 
+static struct zmk_widget_layer_status layer_status_widget;
```

Comment:
We should do this *only* if we have the new Kconfig config enabled, like the other widgets above.